### ISSUE TITLE
cannot change the language

### ISSUE THREAD

**Issue by alexneventchin** [sjarvela/kloudspeaker#143]
I tried to edit the default from en to bg(Bulgarian) then added new file texts_bg.json in localization folders ,but nothing changed.Also even if i edit texts_en.json nothing changes.

UPDATE:Now i can't even see something except white page if i change en to bg in index.html.Mollify works well with this.

**Comment by sjarvela** [comment]
Make sure you have "bg" inside the file also (in field "locale"), not just in the file name.

**Comment by alexneventchin** [comment]
Yep, it's there.I will wait some new version and try to completely install it from scratch.

**Comment by wahmkow** [comment]
I had similar problems with the German translation. Setting up multiple languages to select from did not work. Setting up only German did not work either. I had to use the file with German translations but rename it to English - then it worked. :-(

**Comment by sjarvela** [comment]
Multiple languages sure should work, I haven't been reported any reason why not. On my own environment I have several languages.

One possibility is that some localization files are missing, for example for plugins, but there should be error display for this.

Just to make sure, you have
a) listed all possible languages in index.html initialization
b) set the default language in index.html initialization
c) set user language in admin

Example:

```
$(document).ready(function() {
    require(['kloudspeaker/app'], function(app) {
        app.init({
            "language": {
                options: ["en", "fr", "es"],
                "default": "en"
            },
```

**Comment by alexneventchin** [comment]
I reinstalled KS and now everything is working fine.

Good job.Keep it up.
